icicle-compiler/src/Icicle/Sea/IO/Zebra.hs

```diff
@@ -174,7 +174,6 @@ seaOfDefReadProgram state = vsep
   , "        /* run compute on the facts read so far */"
   , "        if (*fact_count != 0) {"
   , indent 12 $ vsep $ fmap (\i -> pretty (nameOfCompute i) <+> " (program);") computes
-  , "            *fact_count = 0;"
```

Comment:
right, the original condition should really be an invariant that I should prove for the whole program, but we all know that's not gonna happen :)

the `fact_count` is the number of facts read for each iteration of `zebra_read_step`, so it should be cleared there rather than right after compute, as compute is still in the middle of this iteration.

I removed this because I check `fact_count` in the test to find out how many rows I should read back from the icicle input struct.